### DISCUSSION TITLE
json as.. nothing

### DISCUSSION THREAD

**Discussion by ghost** [github-linguist/linguist#6326] [Classification] [answered]
### Additional notes

https://github.com/eladkarako/youtube_to_invidious_redirect_nextgen

![2023-03-16_133239](https://user-images.githubusercontent.com/415238/225604698-a837c696-9536-409c-b655-8cbb03dcd6cc.png)
![2023-03-16_133312](https://user-images.githubusercontent.com/415238/225604703-b531780c-3307-41d5-9022-ad49b05b4680.png)
![2023-03-16_133352](https://user-images.githubusercontent.com/415238/225604705-a3596db8-a9c2-452c-a220-3adfe6af5763.png)

**Comment by lildude** [comment, marked as answer]
This is expected. Once again, as detailed in the "[How Linguist works](https://github.com/github/linguist/blob/master/docs/how-linguist-works.md)" doc:

> Linguist starts by going through all the files in a repository and excludes all files that it determines to be binary data, [vendored code](https://github.com/github/linguist/blob/master/docs/overrides.md#vendored-code), [generated code](https://github.com/github/linguist/blob/master/docs/overrides.md#generated-code), [documentation](https://github.com/github/linguist/blob/master/docs/overrides.md#documentation), or are defined as data (e.g. SQL) or prose (e.g. Markdown) languages, whilst taking into account any [overrides](https://github.com/github/linguist/blob/master/docs/overrides.md).

JSON is considered `data` so is not considered by Linguist when determining the language breakdown of a repo. If you want the JSON to be considered, you'll need to implement the [`linguist-detectable` override](https://github.com/github/linguist/blob/master/docs/overrides.md#detectable).
